oxenss/snode/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+
+add_library(snode STATIC
+    reachability_testing.cpp
+    serialization.cpp
+    service_node.cpp
+    stats.cpp
+    swarm.cpp)
+
+target_link_libraries(snode
+    PUBLIC
+    oxenmq::oxenmq
+    PRIVATE
+    crypto
+    utils
+    server
+    logging
+    rpc
+
```

Comment:
put a comment here saying it is separating internal and external deps instead of a space